server/src/main/java/org/elasticsearch/action/admin/indices/settings/put/UpdateSettingsClusterStateUpdateRequest.java

```diff
@@ -51,4 +53,9 @@ public UpdateSettingsClusterStateUpdateRequest settings(Settings settings) {
         this.settings = settings;
         return this;
     }
+
+    @Override
+    public String toString() {
+        return Arrays.toString(indices()) + settings;
```

Comment:
Should it mention anywhere that this is `UpdateSettingsClusterStateUpdateRequest`? 
I wonder if it might not be immediately obvious what produced the string in case we have many strings from distinct cluster state update tasks.

Reply:
We also include the `source` of the tasks:

https://github.com/elastic/elasticsearch/blob/a733f4287acd7816571cf2d8ad643d47a8d68a28/server/src/main/java/org/elasticsearch/cluster/service/BatchSummary.java#L34

For the update-settings tasks this includes the string `update-settings` so I think there is no chance of confusion here.